### PR TITLE
Revise comments using "we" in WordPress root directory files

### PR DESCRIPTION
**Pull request by ironprogrammer**
- Replaces first-person usage of "we" with second person point of view.
- Also makes small clarification adjustments where voice is much too casual or lacks clear context, especially for non-native English speakers.

This PR focuses on files in the WordPress root directory.

Trac ticket: https://core.trac.wordpress.org/ticket/57052

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/xmlrpc.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
  */
 define( 'XMLRPC_REQUEST', true );
 
-// Some browser-embedded clients send cookies. We don't want them.
+// Some browser-embedded clients send cookies, but are not needed, so discard them.
```
Comment (costdev) [inline src/xmlrpc.php]:
```suggestion
// Discard unneeded cookies sent by some browser-embedded clients.
```
A possible alternative. It changes the original comment more, but might be more direct. What do you think @ironprogrammer?

**Review by costdev — APPROVED** [review]
Just one suggestion that I'm not precious about. LGTM with or without the suggestion :+1:

**Comment by ironprogrammer** [comment]
> Just one suggestion that I'm not precious about. LGTM with or without the suggestion 👍

Thanks, @costdev! Change committed.

File: src/wp-settings.php [hunk 2]
```diff
@@ -70,13 +70,13 @@
 // Standardize $_SERVER variables across setups.
 wp_fix_server_vars();
 
-// Check if we're in maintenance mode.
+// Check if site is in maintenance mode.
```
Comment (SergeyBiryukov) [inline src/wp-settings.php]:
```suggestion
// Check if the site is in maintenance mode.
```

File: src/wp-cron.php [hunk 3]
```diff
@@ -23,7 +23,7 @@
 	header( 'Cache-Control: no-cache, must-revalidate, max-age=0' );
 }
 
-/* Don't make the request block till we finish, if possible. */
+// Don't run cron until request finishes, if possible.
```
Comment (SergeyBiryukov) [inline src/wp-cron.php]:
```suggestion
// Don't run cron until the request finishes, if possible.
```

File: src/wp-login.php [hunk 4]
```diff
@@ -1242,7 +1242,7 @@ function wp_login_viewport_meta() {
 					)
 				);
 			} elseif ( isset( $_POST['testcookie'] ) && empty( $_COOKIE[ TEST_COOKIE ] ) ) {
-				// If cookies are disabled, we can't log in even with a valid user and password.
+				// If cookies are disabled, user can't log in even with a valid username and password.
```
Comment (SergeyBiryukov) [inline src/wp-login.php]:
```suggestion
				// If cookies are disabled, the user can't log in even with a valid username and password.
```

**Comment by ironprogrammer** [comment]
Thanks for the reviews, @SergeyBiryukov! I appreciate it!

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r54866](https://core.trac.wordpress.org/changeset/54866).

**Pull request closed without merging**